client/src/components/Comics.js

```diff
@@ -35,6 +37,12 @@ function Comics() {
     setIsLoading(false);
   };
 
+  function handlePageSize(slider) {
+    console.log(slider.target.value);
+    // setPageSize(slider.target.value);
+    fetchData(slider.target.value);
+  }
+
```

Comment:
```suggestion
  const handleChangePageSize = ({target}) => {
    setPageSize(target.value);
  }

```
I would try to be consistent in using arrow functions vs. regular functions